pkg/endpoints/dashboard.go

```diff
@@ -217,9 +217,9 @@ func getDeployments(r Resource, thingSearchingFor string, namespace string) stri
 }
 
 // Get triggers version
-func GetTriggersVersion(r Resource, isOpenShift bool) string {
+func GetTriggersVersion(r Resource, installedNamespace string, isOpenShift bool) string {
 	version := ""
-	namespacesToCheck := []string{"tekton-pipelines"}
+	namespacesToCheck := []string{installedNamespace}
```

Comment:
Tekton Triggers release yaml is currently hard coded to `tekton-pipelines` so is the most likely place to find it installed
If the dashboard is installed into a different namespace then this wouldn't pick up the triggers version on the About page as it would then look for Triggers in the installed namespace of the dashboard

Reply:
I haven't started working on Triggers helm chart yet, but once it's done the user will be able to choose the target namespace to deploy into.

What do you think about adding a `TRIGGERS_NAMESPACE` environment variable (defaulting to `tekton-pipelines` if not set) ?